### ISSUE TITLE
Localize suggested recipes

### ISSUE THREAD

**Issue by nicmart-dev** [nicmart-dev/feedmenow#39]
Upon changing language with #24 and searching for recipe, the suggested recipes screen #11 shows recipes only in English:
![{3722921D-826D-4A67-98F7-00F30ED658FD}](https://github.com/user-attachments/assets/7d3931ce-dac4-4863-8c14-1b3d7d0ffe76)

Recipes should be using the language defined.

**Comment by yaacovkdev** [comment]
Thank you for the review! I will implement it in the upcoming updates!

**Comment by nicmart-dev** [comment]
I think this issue should be only to localize the output of the LLM from n8n workflow, as we can't rely on react-intl to localize this.
Any issues in static strings still remaining in English (like "Ingredients" or "Directions") are out of scope should be addressed separately, as part of my feedback in #41

**Comment by nicmart-dev** [comment]
I think the best may be, that if the user select a language other than English, we localize recipe (title, ingredients, directions, and overview) on user side on demand each time, using machine translation (so not through n8n which would always return recipe only in English). There appears to be such Node.js package that is still being maintained and appears popular enough: https://www.npmjs.com/package/translate

Things we don't want to do:
- we ask the LLM to provide output in every language (which would be very inefficient and increase recipe search time), not just the one selected by the user, so we can save this and display to the user the matching recipe
- if user changes language, the saved recipes are reset (even though it would avoid this issue and start fresh)

**Comment by nicmart-dev** [comment]
I noticed you made some progress to localize headers for commits like 2fe31a3 or f1aa0fb. 
I noticed the cuisine type is still in English though in the recipe card and recipe overview section:

![Image](https://github.com/user-attachments/assets/b5ea995d-c57b-4e43-8667-521d693926e8)
Let's keep adding those headings to `client/src/i18n/strings.json`

As to the recipe ingredients and directions, did you spend any time investigating that package I mentioned to not rely on n8n for machine translating those on the fly?

**Comment by yaacovkdev** [comment]
> I noticed you made some progress to localize headers for commits like [2fe31a3](https://github.com/nicmart-dev/feedmenow/commit/2fe31a3f6dcebd342c6bee0a824fee754aa94ffb) or [f1aa0fb](https://github.com/nicmart-dev/feedmenow/commit/f1aa0fb29f4f5192455d424fa675c6c00daeef06). I noticed the cuisine type is still in English though in the recipe card and recipe overview section:
> 
> ![Image](https://github.com/user-attachments/assets/b5ea995d-c57b-4e43-8667-521d693926e8) Let's keep adding those headings to `client/src/i18n/strings.json`
> 
> As to the recipe ingredients and directions, did you spend any time investigating that package I mentioned to not rely on n8n for machine translating those on the fly?

Hi Nic, I haven't researched into Translate NPM package yet, but I changed the formatting in localizations. I think it's best to make a separate pull request that would translate (recipe title, ingredients, directions, and overview which includes cuisine name).

**Comment by nicmart-dev** [comment]
Yes agreed, let's address this issue in a separate branch and separate PR.
